### PR TITLE
Changes no row text for containers, images and port forwarding

### LINKED ISSUE #5586: There are no rows to show
The various table views (Images, Port Forwarding, Containers) display the generic text:

> There are no rows to show

They should use the actual item names, e.g.

> There are no images to show
> There are no port forwardings to show
> THere are no containers to show

### PR DESCRIPTION
**Pull request by filipemeneses**
Closes #5586

### CODE REVIEW

File: pkg/rancher-desktop/assets/translations/en-us.yaml [hunk 1]
```diff
@@ -272,6 +276,8 @@ k8s:
     cancel: Cancel
 portForwarding:
   title: Port Forwarding
+  sortableTables:
+    noRows: There are no port forwarding to show
```
Comment (jandubois) [inline pkg/rancher-desktop/assets/translations/en-us.yaml:280]: This sounds a bit weird to me. "Port forwardings" would at least be plural, but still doesn't sound right. How about "port forwarding entries"?

```suggestion
    noRows: There are no port forwarding entries to show
```

Comment (filipemeneses) [reply pkg/rancher-desktop/assets/translations/en-us.yaml:280]: Yea, check spelling pipe flagged this, agree with the suggestion

File: pkg/rancher-desktop/assets/translations/zh-hans.yaml [hunk 2]
```diff
@@ -891,6 +891,20 @@ configmap:
...
       label: 二进制数据
 
+containers:
+  sortableTables:
+    noRows: 没有要显示的容器
+
+images:
+  title: Images
+  sortableTables:
+    noRows: 没有要显示的镜像
```
Comment (mook-as) [inline pkg/rancher-desktop/assets/translations/zh-hans.yaml:901]: (`镜像` appears to be the correct translation, according to [kube docs](https://kubernetes.io/zh-cn/docs/concepts/containers/images/).)

File: pkg/rancher-desktop/assets/translations/zh-hans.yaml [hunk 3]
```diff
@@ -891,6 +891,20 @@ configmap:
     binaryData:
       label: 二进制数据
 
+containers:
+  sortableTables:
+    noRows: 没有要显示的容器
```
Comment (mook-as) [inline pkg/rancher-desktop/assets/translations/zh-hans.yaml:896]: Are these auto-translated? I just find it odd that you're using _可_ (can, may) for the no port forwardings case, which seems a better fit here than _要_ (wanted, desired).

Comment (jandubois) [reply pkg/rancher-desktop/assets/translations/zh-hans.yaml:896]: @vickyhella Can you please double-check the translations! Thanks you!

**Review by jandubois — APPROVED** [review]
Thank you for your contribution! It works fine for me.

I'm just asking our SUSE docs person responsible for the Chinese translations to double-check the strings before I merge it.

File: pkg/rancher-desktop/assets/translations/zh-hans.yaml (hunk 3 above)
Comment (filipemeneses) [reply pkg/rancher-desktop/assets/translations/zh-hans.yaml:896]: Yes, they're auto-translated https://chat.openai.com/share/f615ed08-26c7-4b12-ad7f-c356e0af773b

**Comment by jandubois** [comment]
I'm going to merge this now; translations can always be improved later in a separate PR.

**Pull request merged**